src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashPostBuildCommentAction.java

```diff
@@ -1,21 +0,0 @@
-package stashpullrequestbuilder.stashpullrequestbuilder;
```

Comment:
I'm not sure, but won't removing this class cause warnings about unrecognized data stored in old builds?

Reply:
Actually yes. It started the next day after I installed the plugin with this fix. No idea why it's taking Jenkins so long. That said, I don't think we should keep unused code to prevent those warnings. If there is a nice way to tell Jenkins to suppress those warnings (like "that class is obsolete, don't worry"), I would be happy to incorporate it.

Reply:
I think we need to do a data migration for this, but I'm not sure about specifics. Best I can offer for now is https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility#Hintonretainingbackwardcompatibility-Scenario:renameaclass but this is for renaming a class, not removing it

Reply:
Good, thank you! I think it can be added to `StashPostBuildComment`. That would take some time to test. I'll add a commit for that once I'm sure it's working.

Reply:
OK, I wrote the code, but it doesn't work, and I know why. The `build.xml` files contain information file this:
```
<build>
  <action>
    <stashpullrequestbuilder.stashpullrequestbuilder.StashPostBuildCommentAction plugin="stash-pullrequest-builder@1.9-SNAPSHOT">
        <buildSuccessfulComment>Freestyle is good for 054c04bd78511efd70b6ad82c8c09f973f61bb62</buildSuccessfulComment>
        <buildFailedComment>Freestyle is bad for 054c04bd78511efd70b6ad82c8c09f973f61bb62</buildFailedComment>
    </stashpullrequestbuilder.stashpullrequestbuilder.StashPostBuildCommentAction>
  </action>
</build>
```
Once this PR is applied, there is no other Action that can read that data. StashPostBuildComment is not an Action.

Is that data useful? I don't think so. It's what the plugin would report in case of success and failure. It doesn't tell the user what _actually_ happened. That's a bad design, as it calculates data that may not be needed (I'm actually thinking of adding messages for unstable and aborted builds - unstable builds are actually the ones I want to explain to my users in detail).

I guess we should just accept that the warning would appear. It should be a one time thing. Jenkins will discard the data that should not have been recorded in the first place.

Reply:
> I guess we should just accept that the warning would appear. It should be a one time thing. Jenkins will discard the data that should not have been recorded in the first place.

I agree, cleaning up the build.xml should be an extra benefit